prombench/docs/kind.md

```diff
@@ -0,0 +1,91 @@
+# Prombench in KIND
```

Comment:
Please update the prombench doc and refer to this doc also please include how to delete the cluster locally in the readme. :)